### PR TITLE
[Playwright] - fix payment tests

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Fix payment tests
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | TEST_PATH="functional/BO/10_payment/02_preferences/*" URL_FO=yourShopURL npm run specific-test

### CODE REVIEW

File: tests/puppeteer/pages/BO/payment/preferences/index.js [hunk 1]
```diff
@@ -29,14 +29,15 @@ module.exports = class Preferences extends BOBasePage {
...
+    await this.page.waitForSelector(
+      this.euroCurrencyRestrictionsCheckbox(paymentModule),
+      {state: 'attached'},
     );
     const isCheckboxSelected = await this.isCheckboxSelected(
       this.euroCurrencyRestrictionsCheckbox(paymentModule),
     );
     if (valueWanted !== isCheckboxSelected) {
-      await this.page.click(`${this.euroCurrencyRestrictionsCheckbox(paymentModule)} + i`);
+      await this.page.$eval(`${this.euroCurrencyRestrictionsCheckbox(paymentModule)} + i`, el => el.click());
```
Comment (SimonGrn) [inline tests/puppeteer/pages/BO/payment/preferences/index.js:40]: `click()` is not working ?

Comment (nesrineabdmouleh) [reply tests/puppeteer/pages/BO/payment/preferences/index.js:40]: No is not working because there is + i

Comment (boubkerbribri) [reply tests/puppeteer/pages/BO/payment/preferences/index.js:40]: The checkbox (with id) is not visible 
The `i`  was visible for puppeteer but not playwright
I did this hack for many tests with `#checkbox + i`

Comment (SimonGrn) [reply tests/puppeteer/pages/BO/payment/preferences/index.js:40]: Ok, LGTM then

**Pull request merged**